satellite.py

```diff
@@ -56,6 +60,55 @@
             9: 0.99,
             10: 0.25,
             11: 0.91}
+sampler = neal.Neal()
+
+if len(sys.argv) > 1:
```

Comment:
Instead of hard-coding input data and switching based on CLI arguments, I'd recommend moving the data to separate data files, and modifying the main program to accept the input data file. The default input file might stay `small.json`.

Reply:
I converted both data files to JSON. Great idea, thanks. Much better.